src/main/scala/state/State.scala

```diff
@@ -64,13 +65,14 @@ final case class State(g: Store = Store(),
                        /* TODO: Isn't this data stable, i.e. fully known after a preprocessing step? If so, move it to the appropriate supporter. */
                        predicateSnapMap: Map[ast.Predicate, terms.Sort] = Map.empty,
                        predicateFormalVarMap: Map[ast.Predicate, Seq[terms.Var]] = Map.empty,
-                       isMethodVerification: Boolean = false,
                        retryLevel: Int = 0,
                        /* ast.Field, ast.Predicate, or MagicWandIdentifier */
                        heapDependentTriggers: InsertionOrderedSet[Any] = InsertionOrderedSet.empty,
                        moreCompleteExhale: Boolean = false)
     extends Mergeable[State] {
 
+  val isMethodVerification: Boolean = currentMember.isEmpty || currentMember.get.isInstanceOf[ast.Method]
+
```

Comment:
why do we have the case `currentMember.isEmpty` here? from the name of the variable, I would expect it to be true only when one can show that the current member is actually a `Method`

Reply:
It's None when we're directly verifying a CFG (this functionality is currently unused afaik). I'm pretty sure that for all intents and purposes, that scenario should behave like verifying a method. I'll add a comment.